rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/190_percentiles_hdr_metric.yml

```diff
@@ -418,6 +418,7 @@ setup:
 
   - do:
       search:
+        allow_partial_search_results: true
```

Comment:
Could you explain why this is needed? This test doesn't appear to test partial results?

Reply:
That was a necessity from a previously failing trial on 6.x - if I recall correctly it was for searches that rely on the use of field data. Ditto the other test